src/docs/features/README.md

```diff
@@ -0,0 +1,14 @@
+# Features
```

Comment:
I'm not sure I understand the goal with this new section. Isn't it largely overlapping with the Glossary and Reference sections?

Maybe the Reference section could get a landing page (now it just opens the first page, Content Types) that explains something along the lines of:

```
This is a comprehensive reference for the modules and their features available in Orchard Core. Under each of these pages, you'll find an overview of the given module with documentation on how to use and extend them.
```

But I think providing a categorization of the features based on their areas of function, or goals a user may want to achieve with them is great. So probably also have a similar TOC there as under the "Sections" title here below, and list all related features under the bullet points, linking to their reference pages?

And all the one-sentence summaries under each feature here could be moved to the reference pages, if something like this is missing for the given module.

If you agree, I can create a PR for this.